### PR TITLE
Various changes to improve performance

### PR DESCRIPTION
**Pull request by Roesh**
Try to improve page performance

Tracked in #470

### CODE REVIEW

File: app/core/components/home/VideoPlayer.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import ReactPlayer from "react-player"
+import ReactPlayer from "react-player/lazy"
```
Comment (Roesh) [inline app/core/components/home/VideoPlayer.js:1]: From the (react-player)[https://www.npmjs.com/package/react-player] documentation:
> If your build system supports import() statements, use react-player/lazy to lazy load the appropriate player for the url you pass in. This adds several reactPlayer chunks to your output, but reduces your main bundle size.

File: app/core/components/home/VideoPlayer.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ const VideoPlayer = ({url, className = ""}) => {
         width="100%"
         height="100%"
         controls={true}
+        light={true}
```
Comment (Roesh) [inline app/core/components/home/VideoPlayer.js:12]: From the (react-player)[https://www.npmjs.com/package/react-player] documentation:
> The light prop will render a video thumbnail with simple play icon, and only load the full player once a user has interacted with the image.

File: app/core/components/home/VideoPlayer.js (hunk 1 above)
Comment (Roesh) [reply app/core/components/home/VideoPlayer.js:1]: It also says
> If you only ever use one type, use imports such as react-player/youtube to reduce your bundle size. See config keys for all player keys.
so if we only plan on embedding youtube vids, we could use that instead

**Comment by JuanM04** [comment]
Perfect!
Have you tried using `<Image />`? Is there a performance benefit by using it?

**Comment by Roesh** [comment]
> 
> 
> Perfect!
> Have you tried using `<Image />`? Is there a performance benefit by using it?

Yes, I believe the should be, the typical github image is 460 pixels x 460 pixels and were serving 110 pixels, so we should get a benefit. The issue is that I'm having an error now even though I add the github domains to blitz.config.js; it doesn't seem to be picking that up. I can push those changes and check if the vercel preview is OK

**Comment by JuanM04** [comment]
Well, it doesn't seem to throw an error. The only thing is that the CSS is messed up, but that should be a simple fix

**Comment by Roesh** [comment]
Yep! I didn't expect that lol. Let me try to get my dev environment working and I can make sure the sizing is right

**Comment by JuanM04** [comment]
Perfect, thank you!

**Comment by Roesh** [comment]
So, that works, but it looks like our sizing configuration could be improved. We're still serving the original size, and I notice that there are sizes that are larger present, but not smaller sizes. The smallest size in srcset is 640w, so I'm going to see if this can be made smaller to 128w

**Comment by JuanM04** [comment]
Ok. I don't remember right now (and I'm going to be busy in a few minutes), but maybe we are _only_ using 110px. If that's the case, we could use a fixed size. Could you check that?

Also, btw, could you change the userTrack sponsor image URL to this: `https://raw.githubusercontent.com/blitz-js/blitz/canary/assets/usertrack.png`?

**Comment by Roesh** [comment]
> 
> 
> Ok. I don't remember right now (and I'm going to be busy in a few minutes), but maybe we are _only_ using 110px. If that's the case, we could use a fixed size. Could you check that?
> 
> Also, btw, could you change the userTrack sponsor image URL to this: `https://raw.githubusercontent.com/blitz-js/blitz/canary/assets/usertrack.png`?

Hmm, I don't believe so, on mobile, this changes to 62 px

**Comment by Roesh** [comment]
The only thing that I think may be added here is an error boundary around the image tag just in case we try to get an image from a domain that isn't listed. Actually I'll work on that, might take until end of day since I'll be working on some other stuff, so you can review and merge after that is finished, Juan

**Review by JuanM04 — APPROVED** [review]
:rocket:

**Comment by JuanM04** [comment]
> The only thing that I think may be added here is an error boundary around the image tag just in case we try to get an image from a domain that isn't listed

There is only one image that isn't listed, but it also throws and 401 (it doesn't appear in the README), so... we'll leave it for now

**Pull request merged**